### DISCUSSION TITLE
Newly transferred domain redirecting to Vercel production link, instead of showing project directly

### DISCUSSION THREAD

**Discussion by mikesylla** [vercel/community#5740] [Help]
### Summary

Dear Vercel community,
My domain just got transferred to Vercel from my previous provider, and it seems correctly configured.
However, it's redirecting to the Vercel production link instead of displaying the website directly. (I had set it to redirect before the transfer.)
Do I need to make any adjustments, or is it just a matter of propagation not having taken effect yet?
Best regards,
Michael

### Example

hasfocus.com

### Steps to Reproduce

Simply type the above domain name to a browser.

**Comment by amyegan** [comment]
Hey @michaelpsyllakis. It looks like that domain is going to your site now. Do you still have a problem with it?

**Reply by mikesylla** [reply]
Nope. So it was a propagation thing. I love how fast and responsive Next.js is.

Thanks for looking into this!!!
